Lib/test/test_logging.py

```diff
@@ -4437,8 +4437,11 @@ def _extract_logrecord_process_name(key, logMultiprocessing, conn=None):
             name = mp.current_process().name
 
             r1 = logging.makeLogRecord({'msg': f'msg1_{key}'})
-            del sys.modules['multiprocessing']
-            r2 = logging.makeLogRecord({'msg': f'msg2_{key}'})
+
+            with support.swap_item(sys.modules, 'multiprocessing', None):
+                # We need to restore the previous `multiprocessing` module:
+                # https://bugs.python.org/issue45128
```

Comment:
I don't understand the comment. For me, it's quiet obvious that swap_item() only temporarily overrides `sys.modules['multiprocessing']`. If you want to add a comment, it would be interesting to explain why `sys.modules['multiprocessing']` is set to None, but I don't know the reason. Just remove the comment.

Reply:
👍